examples/data-sources/bloxone_dhcp_hosts/data-source.tf

```diff
@@ -0,0 +1,16 @@
+# Get DHCP Host filtered by an attribute
+data "bloxone_dhcp_hosts" "dhcp_host_by_name" {
```

Comment:
same comment as for the fixed address
`example_by_name`
`example_by_tag`
`example_all`